tests/caasp/stack_conformance.pm

```diff
@@ -41,7 +42,8 @@ sub run {
     # Expect: SUCCESS! -- 123 Passed | 0 Failed | 0 Pending | 586 Skipped PASS
     script_run 'tar -xzf sonobuoy.tgz';
     upload_logs 'plugins/e2e/results/e2e.log';
-    assert_script_run "tail -2 plugins/e2e/results/e2e.log | tee /dev/tty | grep $sb_pass";
+    assert_script_run "tail -10 plugins/e2e/results/e2e.log | tee /dev/tty | grep $sb_pass";
+    assert_script_run "tail -10 plugins/e2e/results/e2e.log | tee /dev/tty | grep $sb_test";
```

Comment:
You should be fine with `tail -5` in first and `tail -1` in second check (but that's up to you).

Please remove `| tee /dev/tty` from second call - output is visible in console from previous command.

Reply:
The reason I've picked a higher number of for the `tail` is because I want to be pro-actively cautious and don't be *borderline* OK. I have the feeling that they might want change something again in the future, which will move this +-1 number of lines.

As for the `| tee /dev/tty`, sure I will remove it.

The verification run: http://skyrim.qam.suse.de/tests/2485#step/stack_conformance/109

Let's wait though about the official answer and discuss next week about this PR.